plugins/pencilblue/controllers/section.js

```diff
@@ -36,10 +36,11 @@ module.exports = function(pb) {
      */
     SectionViewController.prototype.init = function(context, cb) {
         var self = this;
+        var opts = { site: self.site, onlyThisSite: self.onlyThisSite };
         var init = function(err) {
             //get content settings
             var serviceContext = self.getServiceContext();
-            var contentService = new pb.ContentService({site: self.site, onlyThisSite: serviceContext.onlyThisSite});
+            var contentService = new pb.ContentService(opts);
```

Comment:
check the service context to see if it contains the two options you want to pass.